src/Nethermind/Nethermind.Runner/Ethereum/Steps/InitializeBlockchainAuRa.cs

```diff
@@ -273,15 +274,15 @@ protected override TxPool.TxPool CreateTxPool(PersistentTxStorage txStorage)
                 minGasPricesContractDataStore,
                 _api.SpecProvider);
             
-            return new FilteredTxPool(
+            return new TxPool.TxPool(
                 txStorage,
                 _api.EthereumEcdsa,
                 new ChainHeadInfoProvider(_api.SpecProvider, _api.BlockTree, _api.StateReader),
                 NethermindApi.Config<ITxPoolConfig>(),
                 _api.TxValidator,
                 _api.LogManager,
                 CreateTxPoolTxComparer(txPriorityContract, localDataSource),
-                new TxFilterAdapter(_api.BlockTree, txPoolFilter));
+                new TxFilterAdapter(_api.BlockTree, txPoolFilter, _api.LogManager.GetClassLogger<TxFilterAdapter>()));
```

Comment:
Minor: Why not pass logmanager like everywhere else?

Reply:
Minor, there are a few places in the app where we pass LogManager, may be worth to change TxPool so it accepts log manager for filters too but not worth the effort now